app/namespace.go

```diff
@@ -574,6 +574,28 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 						},
 					},
 				},
+				{
+					Name:    "retention",
```

Comment:
for consistency with other commands, we should do ocld namespace retention set --retention-days 30?

Reply:
done

Reply:
[Nit] given that we are specifying a flag on a subcommand called "retention", having "retention" in the flag name and the command seems redundant

Reply:
I will keep this as "retention-days" to allow for usage of the same flag name